bin/release

```diff
@@ -20,8 +23,24 @@ $container = new ContainerBuilder();
 
 $loader = new YamlFileLoader($container, new FileLocator(__DIR__ . '/../config'));
 $loader->load('services.yaml');
+$loader->load('github.yaml');
```

Comment:
For now this is fine, but I think we should find another way to load all yaml files, because the idea of the service container is that `bin/release` does not need knowledge about all the components/services.

Reply:
Maybe we should make a seperate class for that at a later moment.